api/envoy/config/filter/network/redis_proxy/v2/redis_proxy.proto

```diff
@@ -53,6 +53,13 @@ message RedisProxy {
     // need to be known to the cluster manager. If the command cannot be redirected, then the
     // original error is passed downstream unchanged. By default, this support is not enabled.
     bool enable_redirection = 3;
+
+    // Maximum size of buffer before flush is triggered. If this is unset, the buffer flushes
```

Comment:
Can you add a little color on when someone might want to set this vs. not set it, and what a good value would be if they want to try it?